docs/infrastructure/index.asciidoc

```diff
@@ -15,15 +13,58 @@ image::infrastructure/images/infra-sysmon.jpg[Infrastructure Overview in Kibana]
 
 
 [float]
-== Add data sources
-Kibana provides step-by-step instructions to help you add your data sources.
-The {infra-guide}[Infrastructure Monitoring Guide] is a good source for more detailed
-instructions and information.
+== Add data
+
+Kibana provides step-by-step instructions to help you add log data. The
+{infra-guide}[Infrastructure Monitoring Guide] is a good source for more
+detailed information and instructions.
 
 [float]
 == Configure data sources 
 
-By default the Infrastructure UI uses the `metricbeat-*` index pattern to query the data. If you configured Metricbeat to export data to a different set of indices, you will need to set `xpack.infra.sources.default.metricAlias` in `config/kibana.yml` to match your index pattern. You can also configure the timestamp field by overriding `xpack.infra.sources.default.fields.timestamp`. See <<infrastructure-ui-settings-kb>> for a complete list.
+By default the Infrastructure UI uses the `metricbeat-*` index pattern to query
+the data.  If your metrics are located in a different set of indices or use a
+different timestamp field, you can adjust the source configuration via the user
+interface or the {kib} configuration file.
+
+NOTE: The Logs and Infrastructure UI share a common data source definition in
+each space. Changes in one of them can influence the data displayed in the
+other.
+
+[float]
+=== Configure data sources in the user interface
+
+The source configuration UI can be accessed via the corresponding button in the
```

Comment:
I'm wondering if you could get a picture of the gear icon and inline it in this text?
> The source configuration UI can be accessed via the image:gear-icon.png[gear icon alt txt] in the toolbar:

Reply:
I'll try to do that without disrupting the text flow.

Reply:
done:
![grafik](https://user-images.githubusercontent.com/973741/54750503-31adbe80-4bd8-11e9-94c9-c205e7aed71d.png)